tests/def_list/def_list.md

```diff
@@ -0,0 +1,8 @@
+First Term
+: This is the definition of the first term.
+
+Second Term
+: This is one definition of the second term.
```

Comment:
Can you have multiline definitions?

Reply:
Ho yes, my bad I forgot to add a test, and this is actually what does the case you mentionned below